WooCommerce/Classes/ViewRelated/Orders/Order Creation/AddOrderComponentsSection/AddOrderComponentsSection.swift

```diff
@@ -45,10 +45,13 @@ struct AddOrderComponentsSection: View {
             // "Add order components" rows
             Group {
                 addShippingRow
+                    .frame(height: Constants.rowHeight)
 
                 addCouponRow
+                    .frame(height: Constants.rowHeight)
 
                 addGiftCardRow
+                    .frame(height: Constants.rowHeight)
             }
```

Comment:
```suggestion
            Group {
                addShippingRow

                addCouponRow

                addGiftCardRow
            }
            .frame(minHeight: Constants.rowHeight)
```

Reply:
Groups can be used to apply the same modifier to multiple views.

`minHeight` seems more appropriate to allow the buttons to grow with larger dynamic type.

Reply:
> Groups can be used to apply the same modifier to multiple views.

🤦 I knew this one, not sure why I didn't use it. Thanks for updating it!

> minHeight seems more appropriate to allow the buttons to grow with larger dynamic type.

That makes sense!